app/scripts/modules/core/pipeline/config/triggers/travis/travisTrigger.module.ts

```diff
@@ -121,6 +121,6 @@ module(TRAVIS_TRIGGER, [
     formatLabel: (trigger: IBuildTrigger) => {
       return $q.when(`(Travis) ${trigger.master}: ${trigger.job}`);
     },
-    selectorTemplate: '<travis-trigger-options command="vm.command"></travis-trigger-options>',
+    selectorTemplate: require('./selectorTemplate.html'),
```

Comment:
Yep.  that's because the value specified here is being used as the target of an `ng-include`.  See `manualPipelineExecution.html` and `manualPipelineExecution.controller.js`, specifically where `triggerTemplate` is being used.  This is fine as is.